### PR TITLE
Documentation Revisions

### PR DESCRIPTION
**Pull request by AliceBalfanz**
This PR does not change any code, only documentation. 

Things adjusted: 
- made sure the software is called **xcube geoDB** in all occurrences with correct spelling
- made format of Changes and installation.md to suite our best-practice of line length
- fixed text on overview page.

### CODE REVIEW

File: docs/source/installation.md
Comment (AliceBalfanz) [inline docs/source/installation.md:1]: I did not revise it for its correctness only for typos and formatting

File: docs/source/installation.md [hunk 1]
```diff
@@ -62,89 +71,108 @@ $ python setup.py develop
 
 ## 2. Installation of the Database Backend
 
-This section describes how to set up the geoDB PostGIS database. The geoDB PostGIS database consists of three 
+This section describes how to set up the xcube geoDB PostGIS database. The
+xcube geoDB PostGIS database consists of three
 software components:
 
 - A PostGreSQl database (version 14)
 - A PostGIS extension (version 14)
```
Comment (thomasstorm) [inline docs/source/installation.md:79]:
```suggestion
- A PostgreSQL database (version 10)
- A PostGIS extension (version 3+)
```

File: docs/source/overview.rst [hunk 2]
```diff
@@ -1,30 +1,20 @@
...
-analysis-ready form to users. xcube achieves this by carefully converting EO data sources into self-contained *data cubes*
-that can be published in the cloud.
-
-Data Cube
-=========
-
-The interpretation of the term *data cube* in the EO domain usually depends
-on the current context. It may refer to a data service such as `Sentinel Hub`_, to some abstract
-API, or to a concrete set of spatial images that form a time-series.
+xcube geoDB is a geo-spatial database plugin for [xcube](https://github.com/dcs4cop/xcube).
```
Comment (thomasstorm) [inline docs/source/overview.rst:5]:
```suggestion
xcube geoDB is a geo-spatial database within the [xcube](https://github.com/dcs4cop/xcube) ecosystem.
```

File: docs/source/overview.rst [hunk 3]
```diff
@@ -1,30 +1,20 @@
...
+Data Cube (EDC) consortium.
+The processed data can be accessed and queried via a REST API with full querying capabilities.
 
-This section briefly explains the specific concept of a data cube used in the xcube project - the *xcube dataset*.
+================
+Technologies used:
+================
 
+- [xcube](https://github.com/dcs4cop/xcube)
+- [xarray](http://xarray.pydata.org/en/stable/)
```
Comment (thomasstorm) [inline docs/source/overview.rst:16]:
```suggestion
```

**Review by thomasstorm — APPROVED** [review]
Approval if suggestions are followed (or replaced by better suggestions). Otherwise I am very happy about this -- it makes the documentation so much better! Thanks!

**Pull request merged**